src/list/ListItem.tsx

```diff
@@ -30,6 +30,7 @@ export type ListItemProps = TouchableHighlightProps & {
   ViewComponent?: typeof React.Component;
   linearGradientProps?: any;
   children?: any;
+  theme?: FullTheme;
```

Comment:
This shouldn't be necessary. There are other ways to add theme without directly adding it to the component (There are other components that do this)

Reply:
Look at `Input` and how it handles the `theme` prop.

Reply:
I tried the same way, but that doesn't fix the issue, probable reasons are the interface extending other secondary components in the case of ListItem. @flyingcircle

Reply:
fixed @flyingcircle. Please review